resources/views/components/organisms/_o-table-of-contents.blade.php

```diff
@@ -24,7 +24,7 @@
             </div>
         @else
             <span class="m-link-list__item o-accordion__panel-content">
-                <a class="m-link-list__trigger f-secondary {{ $isActive ? 'active' : '' }}" href="{{ $item->url }}"{!! (isset($item->gtmAttributes)) ? ' '.$item->gtmAttributes.'' : '' !!}>
+                <a class="m-link-list__trigger f-secondary {{ $isActive ? 'active' : '' }}" href="{{ $item->present()->getCanonicalUrl }}"{!! (isset($item->gtmAttributes)) ? ' '.$item->gtmAttributes.'' : '' !!}>
```

Comment:
This change can be completely rolled back, because `$item->url` itself calls ` $item->present()->getCanonicalUrl ()`: https://github.com/art-institute-of-chicago/artic.edu/blob/fix/TOC-grouping-url/app/Models/DigitalPublicationArticle.php#L149-L152